pyaerocom/testdata_access.py

```diff
@@ -16,7 +16,8 @@
 from pyaerocom.io import (ReadAeronetSunV3, ReadAeronetSdaV3, ReadAeronetInvV3,
                           ReadEbas, ReadAirNow, ReadGhost, ReadEEAAQEREP_V2)
 
-class TestDataAccess(object):
+class AccessTestData:
```

Comment:
For consistency, please rename module accordingly, to `access_testdata.py`

Reply:
will do